wad_project/templates/socshare/base.html

```diff
@@ -40,13 +40,19 @@
                     <li><a href="{% url 'socshare:register' %}">Register</a></li>
                     {% endif %}
                 </ul>
+                <ul class="list-unstyled nav-links"></ul>
+                    {% if user.is_authenticated %}
+                    <li><a href="{% url 'socshare:logout' %}">Logout</a></li>
+                    {% else %}
+                    {% endif %}
+                </ul>
+
             </nav>
         
             <div class="content {% if fullscreen %}fullscreen{% endif %}">
                 {% if not fullscreen %}
                 <div class="header">
-                    <button onclick="togglesideBar()">AAA</button>
-                    <b>{{ title }}</b>
+                    <button onclick="togglesideBar()" class="btn btn-light"> .. / {{ title }}</button>
```

Comment:
I disagree

Reply:
Why?

Reply:
we decidede to not have ../

also #11 changes that

Reply:
Mainly cause its better than "aaa" and our wireframe had something very similar there but in the end, it's only a placeholder and needs to be changed regardless.

Reply:
fully included all of this in #11 now